torch/serialization.py

```diff
@@ -794,6 +795,9 @@ def load(
             match the :attr:`pickle_module` used to serialize file)
         weights_only: Indicates whether unpickler should be restricted to
             loading only tensors, primitive types and dictionaries
+        mmap: Indicates whether the file should be mmaped rather than loading all the storages into memory.
```

Comment:
nit: I think we could be more explicit on what it does to reduce ambiguity on the exact behavior.
Not sure how to phrase it but the idea would be that we use mmap as a way to go from disk to cpu. Maybe we want to mention that loading is done in two steps: disk -> cpu -> final device (where the second step is a no-op if the final device is cpu).